### ISSUE TITLE
Available contexts check wrongly checking `input` usage in `with` context.

### ISSUE THREAD

**Issue by maikelvdh** [rhysd/actionlint#495]
The following construct is flagged as invalid (`context "inputs" is not allowed here. no context is available here.`):
```
uses: docker://<image-name>:latest
with:
  entrypoint: /bin/sh
  args: >
    --arg "${{ inputs.arg }}"
```

Though based on https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#context-availability this is valid, and equally this works fine.

**Comment by rhysd** [comment]
Thanks for reporting this. This is a bug of actionlint. `inputs` should be allowed be allowed here (`jobs.<job_id>.with.<with_id>`). This is the edge case that `jobs.<job_id>.with.args` cause.

**Comment by maikelvdh** [comment]
Thanks for quick resolution @rhysd! Will you release 1.7.6 to get this availablr by any chance soon?

**Comment by rhysd** [comment]
I'd like to see if some other keys cause similar bugs for a few days. So I'm planning to create the next release at this weekend (Friday or Saturday in JST).

**Comment by Natim** [comment]
IMHO, the 1.7.5 release should have been a 1.8.0 or even a 2.0.0 because it introduces a major change in how `context` is handled.

Once a bug has been discovered, it sounds reasonable not to wait to see if other bugs arrive to release patch releases such as 1.8.1 or 1.7.6 in this case.

This comment doesn't aim to be vindicative but aim to be a reflection around Semantic Versioning and how version number could be less conservative in their incrementation and more communicative of their intentions.

**Comment by rhysd** [comment]
[v1.7.6](https://github.com/rhysd/actionlint/releases/tag/v1.7.6) for fixing this issue was released.
